src/components/Switch/index.tsx

```diff
@@ -1,9 +1,9 @@
-import { VFC } from 'react';
+import { FC } from 'react';
 import { SwitchProps } from '../../types';
 import { StyledSwitch } from './styles';
 
-const Switch: VFC<SwitchProps> = (props) => {
-  return <StyledSwitch {...props} />;
+const Switch: FC<SwitchProps> = ({ className, ...rest }) => {
```

Comment:
Como esse componente não tem children deveria continuar utilizando o `VFC`

Reply:
Pesquisei aqui e no caso é só não usar o `FC` quando não tem `children` ex:

```suggestion
const Switch: SwitchProps = ({ className, ...rest }) => {
```

Reply:
Agora... `SwitchProps = ...` não vai funcionar. Teria que ser `(props: SwitchProps) = ...`.